### ISSUE TITLE
training code

### ISSUE THREAD

**Issue by XAVILLA** [jxhuang0508/CVRN#1]
Hi, thanks for the great work!
Will the training code be released soon?

**Comment by jxhuang0508** [comment]
Hi, currently we are waiting for the patent filing approval.

You could first follow the provided codes to train a baseline model. And then can follow the paper to reproduce the results. Specifically, we first perform cross-view consistency regularization warm-up, i.e., feed a view of one image into network (no gradient) and enforce the prediction of another view to be consistent to this view (grad_back-propagation). And then we perform the cross-view regularized self-training. It is expected to get the results in paper by following these training steps.
